skynet/auth/bearer.py

```diff
@@ -11,5 +11,6 @@ def __init__(self):
     async def __call__(self, request: Request):
         credentials: HTTPAuthorizationCredentials = await super().__call__(request)
 
-        if credentials and await authorize(credentials.credentials):
-            return credentials.credentials
+        request.state.decoded_jwt = await authorize(credentials.credentials)
```

Comment:
You need to check if `credentials` exists, right? If not, auth will fail anyway.